documentation/asset-tracker/topics/assettracker-hardware.md

```diff
@@ -0,0 +1,194 @@
+(assettracker-hardware-doc)=
+
+# AssetTracker hardware
+
+## Supported hardware
+
+The AssetTracker RFID solution is currently certified on [Digitalor](https://www.digitalor.com/en/) asset tracking hardware.
+
+:::{tip}
+Hyperview's automated asset tracking system can be tested and certified on additional asset-tracking hardware solutions. We encourage vendors to contact Hyperview if they are interested in having their asset-tracking hardware system certified.
+:::
+
+The supported hardware models are:
+
+1. The V5008 AIoT Smart Gateway (MQTT) running firmware version 3.x
+2. The V6800 AIoT Gateway (MQTT) running firmware version 1.x
+
+Please [contact](https://www.hyperviewhq.com/contact/) the Hyperview Sales or Support teams for more information.
+
+## Installation
+
+### Selecting a hardware Configuration
+
+AssetTracker can be installed in two different configurations:
+
+1. One-to-One: One Expansion Hub to one Gateway (V5008 AIoT)
+2. Daisy Chain: Up to 5 Expansion Hubs to one Gateway (V5008 AIoT)
+3. One to many: up to 24 Expansion Hubs to one Gateway (V6800 AIoT)
+
+```{figure} /asset-tracker/media/assettracker_diagram2.png
+:alt: AssetTracker deployment diagram 3
+:class: border-black
+
+Example Daisy Chain Deployment
+```
+
+For installing AssetTracker in multiple racks in a row, you may daisy-chain up to five Expansion Hubs to a single gateway. The maximum length of the cables from the Gateway to the last Expansion Hub in the chain must not exceed _20 meters (65 feet)_.
+
+Each Master Module has an address between 1 (one) and 5 (five). Each Master Module must have a _unique_ address in the Daisy Chain configuration. The address specification is printed on the Master Module tag.
+
+```{figure} /asset-tracker/media/master_modules_1.png
+:alt: Master Modules image 1
+:class: border-black
+
+Master Modules with ID and address (Circled)
+```
+
+### Network requirements
+
+:::{important}
+You must [contact our support team](https://system.hyperviewhq.com/helpdesk) to get access to the latest vendor configuration software and documentation.
+:::
+
+- The Gateway uses port 5656/TCP for management and configuration. The configuration must be done using the supplied tool. It is strongly recommended that the tool be one the same network segment as the gateway to minimize network related issues. A copy of the configuration tool will be provided as part of the license activation and onboarding process.
```

Comment:
Do you mean "...on the same..." instead of "...one the same..."?

Reply:
Thanks! It is fixed.